MIGRATION.md

```diff
@@ -1,5 +1,13 @@
 # Deprecations in 7.x
 
+You can use [@sentry/migr8](https://github.com/getsentry/sentry-migr8) to automatically update your SDK usage and fix most deprecations:
+
+```bash
+npx @sentry/migr8
```

Comment:
```suggestion
npx @sentry/migr8@latest
```

This is usually a good idea for these wizard type things. You don't want users to run a cached version.